[2024-06-16T04:24:11.553+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dust_data_pipeline.ETL.populate_office_table manual__2024-06-16T04:23:45.794193+00:00 [queued]>
[2024-06-16T04:24:11.568+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dust_data_pipeline.ETL.populate_office_table manual__2024-06-16T04:23:45.794193+00:00 [queued]>
[2024-06-16T04:24:11.568+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2024-06-16T04:24:11.592+0000] {taskinstance.py:1327} INFO - Executing <Task(PythonOperator): ETL.populate_office_table> on 2024-06-16 04:23:45.794193+00:00
[2024-06-16T04:24:11.599+0000] {standard_task_runner.py:57} INFO - Started process 227 to run task
[2024-06-16T04:24:11.607+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dust_data_pipeline', 'ETL.populate_office_table', 'manual__2024-06-16T04:23:45.794193+00:00', '--job-id', '298', '--raw', '--subdir', 'DAGS_FOLDER/test_redshift_real_v3.py', '--cfg-path', '/tmp/tmpsrtze4h7']
[2024-06-16T04:24:11.610+0000] {standard_task_runner.py:85} INFO - Job 298: Subtask ETL.populate_office_table
[2024-06-16T04:24:11.685+0000] {task_command.py:410} INFO - Running <TaskInstance: dust_data_pipeline.ETL.populate_office_table manual__2024-06-16T04:23:45.794193+00:00 [running]> on host 2c7bc140ebc2
[2024-06-16T04:24:11.820+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dust_data_pipeline' AIRFLOW_CTX_TASK_ID='ETL.populate_office_table' AIRFLOW_CTX_EXECUTION_DATE='2024-06-16T04:23:45.794193+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-16T04:23:45.794193+00:00'
[2024-06-16T04:24:11.838+0000] {base.py:73} INFO - Using connection ID 'redshift_test_dev' for task execution.
[2024-06-16T04:24:11.961+0000] {sql.py:375} INFO - Running statement: 
    DROP TABLE IF EXISTS yusuyeon678.group_by_office;
    CREATE TABLE yusuyeon678.group_by_office (
        date TIMESTAMP NOT NULL,
        office_code INT NOT NULL,
        office_name VARCHAR(20) NOT NULL,
        dust FLOAT NOT NULL,
        ultradust FLOAT NOT NULL,
        O3 FLOAT NOT NULL,
        NO2 FLOAT NOT NULL,
        CO FLOAT NOT NULL,
        SO2 FLOAT NOT NULL,
        CONSTRAINT group_by_office_pk PRIMARY KEY (date, office_code)
    );
    , parameters: None
[2024-06-16T04:24:12.133+0000] {base.py:73} INFO - Using connection ID 'redshift_test_dev' for task execution.
[2024-06-16T04:24:12.290+0000] {sql.py:375} INFO - Running statement: 
    INSERT INTO yusuyeon678.group_by_office (date, office_code, office_name, dust, ultradust, O3, NO2, CO, SO2)
    SELECT
        date,
        office_code,
        office_name,
        ROUND(AVG(dust), 2) AS dust,
        ROUND(AVG(ultradust), 2) AS ultradust,
        ROUND(AVG(o3), 2) AS o3,
        ROUND(AVG(no2), 2) AS no2,
        ROUND(AVG(co), 2) AS co,
        ROUND(AVG(so2), 3) AS so2
    FROM yusuyeon678.raw_data
    GROUP BY date, office_code, office_name;
    , parameters: None
[2024-06-16T04:24:12.596+0000] {sql.py:384} INFO - Rows affected: 400
[2024-06-16T04:24:12.820+0000] {python.py:183} INFO - Done. Returned value was: None
[2024-06-16T04:24:12.843+0000] {taskinstance.py:1350} INFO - Marking task as SUCCESS. dag_id=dust_data_pipeline, task_id=ETL.populate_office_table, execution_date=20240616T042345, start_date=20240616T042411, end_date=20240616T042412
[2024-06-16T04:24:12.903+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 0
[2024-06-16T04:24:12.959+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
